server/server.js

```diff
@@ -21,35 +22,50 @@ app.use(bodyParser.urlencoded({
 app.use('/static', express.static(path.join(__dirname, '/static')));
 
 
+
+var bobList = [];
+
+
+function nowPlusXMinutes(numMinutes) {
```

Comment:
We're not using it right now, so I removed it, and we'll find a library when the time comes to add that functionality to the frontend.